__docs__/wonder-blocks-theming/tokens-border.mdx

```diff
@@ -0,0 +1,96 @@
+import {Meta, Source} from "@storybook/blocks";
+
+import TokenTable from "../../.storybook/components/token-table";
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+<Meta title="Theming / Tokens / Border" />
+
+export const baseColumns = (kind) => [
```

Comment:
**Optional suggestion:** Can we use `property` instead of `kind` here? I think that might make it a little more clear about how this relates to the border. Same with all the other `baseColumns` constants declared in the other files.